src/exceptions.rs

```diff
@@ -609,9 +610,8 @@ mod test {
         let py = gil.python();
         let err = py
             .run("raise Exception('banana')", None, None)
-            .expect_err("raising should have given us an error")
-            .instance(py);
-        write!(&mut out, "{}", err).expect("successful format");
+            .expect_err("raising should have given us an error");
+        write!(&mut out, "{}", err.instance(py)).expect("successful format");
```

Comment:
if `PyErr` now implements Display, shouldn't that work without the `instance` call?